# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1370840110 -28800
#      Mon Jun 10 12:55:10 2013 +0800
# Node ID e3c931e5b707c35e6fddb6672cfc07c45b3f4e4b
# Parent  55ba77da26d30f85c7bd2b7cb773521f3e3a883c
Modify the "Workspace" tab to properly display the work space colors on MacOS X.

* libgui/src/settings-dialog.ui: Place the "Storage Class Colors" QGroupBox
inside a QVBoxLayout.  Add horizontal lines for their esthetic effect.
The settings-dialog.ui had to be manually edited to remove the font property
for the QGroupBox (Qt Designer bug?).

diff --git a/libgui/src/settings-dialog.ui b/libgui/src/settings-dialog.ui
--- a/libgui/src/settings-dialog.ui
+++ b/libgui/src/settings-dialog.ui
@@ -1065,35 +1065,93 @@
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_workspace">
       <attribute name="title">
        <string>Workspace</string>
       </attribute>
-      <widget class="QGroupBox" name="workspace_colors_box">
+      <widget class="QWidget" name="verticalLayoutWidget_6">
        <property name="geometry">
         <rect>
-         <x>19</x>
-         <y>19</y>
-         <width>631</width>
-         <height>81</height>
+         <x>0</x>
+         <y>8</y>
+         <width>671</width>
+         <height>391</height>
         </rect>
        </property>
-       <property name="minimumSize">
-        <size>
-         <width>0</width>
-         <height>81</height>
-        </size>
-       </property>
-       <property name="title">
-        <string>Storage Class Colors</string>
-       </property>
+       <layout class="QVBoxLayout" name="verticalLayout_13">
+        <item>
+         <widget class="Line" name="line_7">
+          <property name="minimumSize">
+           <size>
+            <width>0</width>
+            <height>0</height>
+           </size>
+          </property>
+          <property name="orientation">
+           <enum>Qt::Horizontal</enum>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QGroupBox" name="workspace_colors_box">
+          <property name="enabled">
+           <bool>true</bool>
+          </property>
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+            <horstretch>0</horstretch>
+            <verstretch>0</verstretch>
+           </sizepolicy>
+          </property>
+          <property name="minimumSize">
+           <size>
+            <width>0</width>
+            <height>81</height>
+           </size>
+          </property>
+          <property name="title">
+           <string>Storage Class Colors</string>
+          </property>
+          <zorder>line_6</zorder>
+          <zorder>line_5</zorder>
+         </widget>
+        </item>
+        <item>
+         <widget class="Line" name="line_8">
+          <property name="minimumSize">
+           <size>
+            <width>0</width>
+            <height>1</height>
+           </size>
+          </property>
+          <property name="orientation">
+           <enum>Qt::Horizontal</enum>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <spacer name="verticalSpacer_6">
+          <property name="orientation">
+           <enum>Qt::Vertical</enum>
+          </property>
+          <property name="sizeHint" stdset="0">
+           <size>
+            <width>20</width>
+            <height>40</height>
+           </size>
+          </property>
+         </spacer>
+        </item>
+       </layout>
       </widget>
+      <zorder>verticalLayoutWidget_6</zorder>
+      <zorder>line_6</zorder>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Network</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_5">
